### PR TITLE
Added support for Swift 2.3

### PR DESCRIPTION
**Pull request by bonkey**
#107: Added support for Swift 2.3

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutTransitioningDelegate.swift [hunk 1]
```diff
@@ -6,10 +6,24 @@ import UIKit
...
+    return CheckoutPresentationController(presentedViewController: presented, presentingViewController: presenting)
+    }
+
+    #else
+
     func presentationControllerForPresentedViewController(presented: UIViewController,
-        presentingViewController presenting: UIViewController,
-        sourceViewController source: UIViewController) -> UIPresentationController? {
-        return CheckoutPresentationController(presentedViewController: presented, presentingViewController: presenting)
+                                                          presentingViewController presenting: UIViewController,
```
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/CheckoutTransitioningDelegate.swift]: indentation is destroyed, that's why I prefer one line for method definitions  
especially it's apple method :P

**Comment by hani-ibrahim** [comment]
buddy build error

File: AtlasUI/AtlasUI/CheckoutTransitioningDelegate.swift (hunk 1 above)
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutTransitioningDelegate.swift]: Well, I can say one thing: I'm seeing code formatting issues always.

And everywhere.

Unless a project has versatile auto-formatter.

[Swimat](https://github.com/Jintin/Swimat) is far from perfect, but still pretty nice, `./calypso.rb clean all` does some job as well.

**Comment by bonkey** [comment]
xUnique combined with merging strategy destroyed pbxproj files... I'll sit over it tomorrow or sth.

**Comment by bonkey** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**